### PR TITLE
{lang}[intel/2021b] SciPy-bundle v2021.10 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2021.10-intel-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    }),
+    ('gast', '0.5.2', {
+        'checksums': ['f81fcefa8b982624a31c9e4ec7761325a88a0eba60d36d1da90e47f8fe3c67f7'],
+    }),
+    ('beniget', '0.4.1', {
+        'checksums': ['75554b3b8ad0553ce2f607627dad3d95c60c441189875b98e097528f8e23ac0c'],
+    }),
+    ('pythran', '0.10.0', {
+        'checksums': ['9dac8e1d50f33d4676003e350b1f0c878ce113e6f907920e92dc103352cac5bf'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2021.10-intel-2021b.eb:53]: If scipy fails to build with pythran when using intel compilers, should we be removing these items that were only added to support building scipy with pythran? Though that would then make this inconsistent with the foss version.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3118.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/860322d9baf952fca594e1a4ca3fe0cf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2616.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/2ae3718e0be2e26d9e3be527e821a9b0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3519.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/ccd7e54b3ab251d41d6c93167c392256 for a full test report.

File: easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2021.10-intel-2021b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2021.10-intel-2021b.eb:53]: Yeah, wasn't sure here either...

It's not totally broken though, Pythran itself seems functional (cfr. https://github.com/serge-sans-paille/pythran#basic-usage):

```shell
$ module load SciPy-bundle/2021.10-intel-2021b

$ pythran dprod.py

$ ls dprod.*
dprod.cpython-39-x86_64-linux-gnu.so  dprod.py

$ python -c 'import dprod; print(dprod.__file__); print(dprod.dprod(list(range(1, 1000)), list(range(1001, 2000))))'
/tmp/dprod.cpython-39-x86_64-linux-gnu.so
832333500
```

So I'd rather leave it in, perhaps a patch turns out that we can apply easily to build `scipy` with Pythran support later...

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/4ec77e38c1ce50374cb8a9a91e34d80c for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0104u09a.bear.cluster - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/9757e2d91b3669b849b30486851f5742 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/9a785120f8ee794248c3b5a4e6d79b12 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jsfc01.int.jusuf.sebastian.cluster - Linux rocky linux 8.4, x86_64, AMD EPYC 7742 64-Core Processor (zen2), 1 x NVIDIA GRID V100-4C, 460.73.01, Python 3.6.8
See https://gist.github.com/a7093cb3c44d7f0988a2761bac2b0e50 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**